app/server/appsmith-server/src/main/java/com/appsmith/server/domains/ApplicationJson.java

```diff
@@ -0,0 +1,26 @@
+package com.appsmith.server.domains;
+
+import com.appsmith.external.models.DecryptedSensitiveFields;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+@Getter
```

Comment:
Again, please confirm/fix my understanding 👇

```suggestion
/**
 * A DTO class to hold complete information about an application, which will then be serialized to a file so as to
 * export that application into a file.
 */
@Getter
```

Reply:
Comments are right on the spot!